overlord/state/change.go

```diff
@@ -28,12 +28,15 @@ type Status int
 
 // Admitted status values for changes and tasks.
 const (
-	Running Status = 0
-	Waiting Status = 1
-	Done    Status = 2
-	Error   Status = 3
+	StatusDefault Status = 0
```

Comment:
Sorry for the bogus suggestion earlier, but can we please have this as `DefaultStatus` instead?

Also, I think we should suffix all of the other ones, otherwise we'll end up with things like state.Error, which looks like something else.